e2e_tests/conftest.py

```diff
@@ -115,3 +127,37 @@ def temporary_s3_bucket():
     create_s3_bucket_if_not_exists(bucket_name, 'us-west-2')
     yield f's3://{bucket_name}/repo'
     _cleanup_s3_bucket_if_exist(bucket_name, 'us-west-2')
+
+
+@pytest.fixture(scope='session')
+def temporary_yatai_service_url():
+    ensure_docker_available_or_raise()
+    container_name = f'e2e-test-yatai-service-container-{uuid.uuid4().hex[:6]}'
+    yatai_service_url = 'localhost:50051'
+
+    command = [
+        'docker',
+        'run',
+        '--rm',
+        '--name',
+        container_name,
+        '-p',
+        '50051:50051',
+        '-p',
+        '3000:3000',
+        f'bentoml/yatai-service:{PREV_PYPI_RELEASE_VERSION}',
```

Comment:
it should use the local code repo to install bentoml right? otherwise, this is just testing the last pypi release

```
docker build -<<EOF
FROM bentoml/yatai-service:{PREV_PYPI_RELEASE_VERSION}
ADD . /bentoml-local-repo
RUN pip install /bentoml-local-repo
EOF
```